File: src/main/java/com/morak/performancetracker/context/Context.java
package com.morak.performancetracker.context;

import java.util.List;

public class Context {

    private final String name;
    private final List<Scope> scopes;

    public Context(String name, List<Scope> scopes) {
        this.name = name;
        this.scopes = scopes;
    }

    public List<Scope> getScopes() {
        return scopes;
    }

    public String getName() {
        return this.name;
    }
}

File: src/main/kotlin/com/morak/performancetracker/context/Context.kt
package com.morak.performancetracker.context

class Context {
    val name: String?
    val scopes: List<Scope>

    constructor(scopes: List<Scope>) {
        name = null
        this.scopes = scopes
    }

    constructor(name: String?, scopes: List<Scope>) {
        this.name = name
        this.scopes = scopes
    }
}
